static/app/views/issueList/searchBar.spec.tsx

```diff
@@ -165,4 +166,68 @@ describe('IssueListSearchBar', function () {
       );
     });
   });
+
+  describe('Tags and Fields', function () {
+    it('displays the correct options for the is tag', async function () {
+      MockApiClient.addMockResponse({
+        url: '/organizations/org-slug/tags/',
+        body: [],
+      });
+
+      defaultProps.organization.features = ['issue-stream-search-query-builder'];
+
+      render(<IssueListSearchBar {...defaultProps} />, {
+        router,
+      });
+
+      await userEvent.click(screen.getByRole('combobox', {name: 'Add a search term'}));
+      await userEvent.paste('is:', {delay: null});
+      await userEvent.click(
+        screen.getByRole('button', {name: 'Edit value for filter: is'})
+      );
+
+      Object.values(IsFieldValues).forEach(value => {
+        expect(screen.getByRole('option', {name: value})).toBeInTheDocument();
+      });
+    });
+
+    it('displays the correct options under Issue Filters', async function () {
+      MockApiClient.addMockResponse({
+        url: '/organizations/org-slug/tags/',
+        body: [{key: 'someTag', name: 'Some Tag'}],
+      });
+
+      defaultProps.organization.features = ['issue-stream-search-query-builder'];
+
+      render(<IssueListSearchBar {...defaultProps} />, {
+        router,
+      });
+
+      await userEvent.click(screen.getByRole('combobox', {name: 'Add a search term'}));
+      await userEvent.click(screen.getByRole('button', {name: 'Event Tags'}));
+
+      expect(screen.getByRole('option', {name: 'someTag'})).toBeInTheDocument();
```

Comment:
This expectation you really want to wait for since there is an async request being made